Docs: Updated 12-factor tutorials (#2251)
canonical/charmcraft: docs/tutorial/kubernetes-charm-django.rst

    A rock is a special kind of OCI-compliant container image, while a
    charm is a software operator for cloud operations that use the Juju
    orchestration engine. The result is Django apps that

>     can be deployed, configured, scaled, integrated, etc.,

Reviewer: While we're editing this line, can we do this too? :)

```suggestion
    can be deployed, configured, scaled, integrated, and so on,
```

Let's start by creating the "Hello, world" Django app that
will be used for this tutorial.

> Create and open a new file ``requirements.txt`` in a text editor using
> ``nano requirements.txt``, copy the following text into it and then
> save the file:

Reviewer: Following the advice about functional and mechanical actions, I think this could be even clearer:

```suggestion
Create a new requirements file with ``nano requirements.txt``.
Then, copy the following text into it, and save:
```

Also consider that for this audience, we probably don't need to specify a particular text editor. I think for many readers the mention of nano would be distracting, since it's usually an editor of last resort.

We could just say:

> Create a `requirements.txt` file and copy the following text into it:

Author: Specifying a specific text editor was a request from several UX sessions. For a part of our user base (mostly students, the "Novice User" as we've named this particular user persona), they don't have background knowledge on using terminal text editors. Without being specific about a terminal text editor, the Novice User gets immediately stuck and cannot continue without assistance. I have some recorded UX sessions where this happens, and I can share them with you if you'd like.

I'm aware that it's probably a minority of the users, and I know that `nano` isn't everyone's preferred choice of terminal text editor (it isn't mine, for instance) but I do feel strongly like we need to provide *something* specific here.

Reviewer: Do you think the original suggestion would work?

Author: The first suggestion will work. I'll update now, thanks :)
